### DISCUSSION TITLE
Disable Git LFS organization-wide on GitHub.com

### DISCUSSION THREAD

**Discussion by sa-spag** [community/community#23656] [Enterprise]
<p>Because Git Large File Storage (LFS) usage is subject to <a href="https://docs.github.com/en/free-pro-team@latest/github/managing-large-files/about-storage-and-bandwidth-usage" rel="noopener nofollow ugc">quotas on GitHub.com</a> and because there are alternatives to Git LFS, disabling Git LFS at the organization level is a legitimate policy to be applied by some administrator. It is <a href="https://docs.github.com/en/enterprise-server@2.20/admin/user-management/configuring-git-large-file-storage-for-your-enterprise#configuring-git-large-file-storage-for-your-enterprise" rel="noopener nofollow ugc">possible to do so on self-hosted instances</a> but apparently not on <a href="http://GitHub.com" rel="noopener nofollow ugc">GitHub.com</a>.</p>

**Comment by canuckjacq** [comment, marked as answer]
<p>We don’t have it as a self-service option but it looks like it’s possible to do on our end.</p>
<p>To request that, please <a href="https://support.github.com/contact">open a ticket</a> and make sure you specify the names of the organisation/s you’d like this done for. They’ll be able to sort that out for you provided you’re the organisation owner.</p>

**Reply by mrmanc** [reply]
Will this disable LFS in repos where it is already being used? I’d like to prevent any _**more**_ repos using LFS, but not impact those that already do.

**Comment by usmonster** [comment]
<aside class="quote group-github-employees" data-username="canuckjacq" data-post="2" data-topic="141422">
<div class="title">
<div class="quote-controls"></div>
 canuckjacq:</div>
<blockquote>
<p>We don’t have it as a self-service option but it looks like it’s possible to do on our end.</p>
</blockquote>
</aside>
<p>Does this mean it will never be a self-service option for Organization/Enterprise Cloud owners, or is there an internal ticket to eventually add this? I’d love to see this on the roadmap.</p>

**Comment by canuckjacq** [comment]
<p>It definitely doesn’t mean never. For actions that are only available to support, the move to self service is usually determined by demand.</p>

**Comment by usmonster** [comment]
<p>Thanks for the fast and hopeful response! I guess demand is tracked by support requests or an internal ticket? I wondered if there was a more public way to express &amp; gather interest in this feature. (I was using my :heart: on this thread to do so, but I doubt anyone else will see this now that it’s closed.) If opening a support request is the only way, I will do that. Thanks again!</p>
